app/Http/Requests/TestRequest.php

```diff
@@ -24,13 +24,15 @@ public function authorize()
     public function rules()
     {
         return [
-            'name' => 'required|regex:/^[A-Za-z0-9,\-\.]+$/|max:50',
-            'information' =>'max:300',
-            'phone_number'=>'required|regex:/^[\(\)\-\.\+\/0-9]+$/|max:20',
-            'birthday'=>'required|date|before:yesterday|after:60 year ago',
-            'position_id'=>'required',
-            'avatar'=>'required|mimes:png,jpeg,gif|max:10240',
-            'gender'=>'required|in:"male","female"'
+            'name'=>'regex:/^[A-Za-z0-9,\-\.]+$/|max:10',
```

Comment:
What is the purpose of this file? It does not seem to be used?

Reply:
i validate name is  alphanumeric,, include character and number

Reply:
Yes but I'm asking about the whole file. I see that you do not use the class `TestRequest` anywhere in the code.